exercises/exercise2.js

```diff
@@ -19,13 +40,25 @@ module.exports.botScripts = [
   // mychatbot responds to user "The area is 78.5398...."
   
 
-
-
+  
+  
 
   // Write an interaction where one user asks the bot 
   // "Tell me a story with NAME1 and NAME2" where NAME1 and NAME2 is any string
   // The chatbot will respond with the answer.
   // 
   // mychatbot> mychatbot Tell me a story with Jack and Jill
-  // mychatbot responds to user "Jack and Jill went up the hill to fetch a pail of water."
+  // mychatbot responds to user "Jack and Jill went up the hill to fetch a pail of water
+  
+  {
+    label: 'Tell Story', 
+  prompt: "Tell me a story with #{NAME} and #{NAME}",
```

Comment:
Since you are calling 'NAME1' and 'NAME2' in the function for this prompt, make sure to include the 1 and 2 in your params in the prompt. ex: #{NAME1} ... #{NAME2}